packages/runtime/src/ax.tsx

```diff
@@ -19,23 +19,22 @@
  *
  * @param classes
  */
-export const ax = (classes: (string | undefined | false)[]): string => {
-  const found: Record<string, string> = {};
+export default function ax(classNames: (string | undefined | false)[]): string {
+  const atomicGroups: Record<string, string> = {};
+  let i = -1;
 
-  for (let i = 0; i < classes.length; i++) {
-    const cls = classes[i];
-    if (!cls) {
+  while (++i < classNames.length) {
+    if (!classNames[i]) {
       continue;
     }
 
-    const groups = cls.split(' ');
+    const groups = (classNames[i] as string).split(' ');
+    let x = -1;
 
-    for (let x = 0; x < groups.length; x++) {
-      const className = groups[x];
-      const group = className.slice(0, className.charCodeAt(0) === 95 ? 5 : undefined);
-      found[group] = className;
+    while (++x < groups.length) {
+      atomicGroups[groups[x].slice(0, 5)] = groups[x];
     }
   }
 
-  return Object.values(found).join(' ');
-};
+  return Object.values(atomicGroups).join(' ');
```

Comment:
Q:  We are targeting `es5` right 🤔? This is for `Object.values`.

Reply:
its supported in everything except for IE11 which is okay